### PR TITLE
Simple Logging

### PR DESCRIPTION
**Pull request by Shivansh-007**

### CODE REVIEW

File: PyLudus/__init__.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+from __future__ import annotations
+
+import os
+import typing
+
+import loguru
```
Comment (Pyroseza) [inline PyLudus/__init__.py]: why loguru? can we not just use the built in logging module?
https://docs.python.org/3/library/logging.html
https://docs.python.org/3/howto/logging.html

Comment (Shivansh-007) [reply PyLudus/__init__.py]: I feel `loguru` is much simpler to use and is better than the standard library `logging`.

Comment (Pyroseza) [reply PyLudus/__init__.py]: I’m not convinced, I feel like we should only go for extra modules if there is some sort of competitive advantage. Logging is not a complicated thing and adding a whole new module for barely any extra gain to me is not worth it, django has some nice docs on how simple and flexible logging is in Python out of the box: https://docs.djangoproject.com/en/3.1/topics/logging/

Comment (Shivansh-007) [reply PyLudus/__init__.py]: Resolved in commit b11156d16ad092f38be6419e841db44bf8738909.

**Comment by Pyroseza** [comment]
@Shivansh-007 looks like the logging in dash is not picking the logging config in settings

**Comment by Shivansh-007** [comment]
> @Shivansh-007 looks like the logging in dash is not picking the logging config in settings

Not sure why this happening.

**Review by Shivansh-007 — COMMENTED** [review]
Works :shipit:

**Pull request merged**